frontend/src/pages/meditation/components/meditation-sidebar/meditation-sidebar.tsx

```diff
@@ -0,0 +1,110 @@
+import meditationPlaceholder from '#assets/img/meditation-image-placeholder.png';
+import {
+  Button,
+  Card,
+  Search,
+  Sidebar,
+  SidebarBody,
+  SidebarHeader,
+} from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import {
+  useAppDispatch,
+  useCallback,
+  useRef,
+  useSearch,
+} from '#libs/hooks/hooks.js';
+import { actions as meditationActions } from '#slices/meditation/meditation.js';
+
+import { MeditationModal } from '../meditation-modal/meditation-modal.js';
+import styles from './styles.module.scss';
+
+const mockedMeditations = [{ id: 1, name: 'Meditation' }];
+
+const mockedSelectedMeditation = {
+  id: 1,
+};
+
+type Properties = {
+  isSidebarShown: boolean;
+  setIsSidebarShown: (value: boolean) => void;
+};
+
+const MeditationSidebar: React.FC<Properties> = ({
+  isSidebarShown,
+  setIsSidebarShown,
+}) => {
+  const dispatch = useAppDispatch();
+  const { filteredElements, setFilter } = useSearch(mockedMeditations, 'name');
+  const dialogReference = useRef<HTMLDialogElement>(null);
+
+  const handleOpen = useCallback(() => {
+    dialogReference.current?.showModal();
+  }, [dialogReference]);
+
+  const handleSubmit = useCallback(
+    (title: string, file: File) => {
+      void dispatch(
+        meditationActions.createMeditationEntry({
+          file: file,
+        }),
+      );
+    },
+    [dispatch],
+  );
+
+  const handleSelectChat = useCallback(
+    (id: number) => {
+      return () => {
+        mockedSelectedMeditation.id = id;
+        setIsSidebarShown(false);
+        // TODO redux logic
+      };
+    },
+    [setIsSidebarShown],
+  );
```

Comment:
we don't need this
```suggestion
```

Reply:
@Liza-Veis I left it only with `setIsSidebarShown(false);` in order to handle low width page display. Should I forget about it for now and totally remove it?

Reply:
yes, you can leave it with setIsSidebarShown